agilex_piper/piper.xml

```diff
@@ -0,0 +1,300 @@
+            <geom mesh="link3_27" material="gray_mat" class="visual" />
+            <geom mesh="link3_28" material="gray_mat" class="visual" />
+
+            <geom mesh="link3" class="collision" />
+
+            <body name="link4"
+              pos="-0.24221 0.068514 0" quat="0.544767 -0.544769 -0.450809 0.450808">
+              <inertial pos="0.000276465 -0.00102804 -0.00472831" quat="0.492305 0.505285 0.503294 0.499017"
+                mass="0.127087" diaginertia="4.87057e-05 4.10595e-05 3.82001e-05" />
+              <joint name="joint4" pos="0 0 0" axis="0 0 1" range="-1.832 1.832" />
+              <geom mesh="link4_2" material="gray_mat" class="visual" />
+              <geom mesh="link4_3" material="darker_gray_mat" class="visual" />
+              <geom mesh="link4_4" material="gray_mat" class="visual" />
+              <geom mesh="link4_5" material="gray_mat" class="visual" />
+              <geom mesh="link4_6" material="gray_mat" class="visual" />
+              <geom mesh="link4_2" class="collision" />
+              <geom mesh="link4_3" class="collision" />
+              <geom mesh="link4_4" class="collision" />
+              <geom mesh="link4_5" class="collision" />
+              <geom mesh="link4_6" class="collision" />
+
+              <body name="link5" quat="0.707105 0.707108 0 0">
+                <inertial pos="8.82262e-05 0.0566829 -0.0019612" quat="0.507447 0.490986 0.506887 0.494466"
+                  mass="0.144711" diaginertia="0.001 0.001 0.001" />
+                <joint name="joint5" pos="0 0 0" axis="0 0 1" range="-1.22 1.22" />
+                <geom mesh="link5_2" material="black_mat" class="visual" />
+                <geom mesh="link5_3" material="black_mat" class="visual" />
+                <geom mesh="link5_4" material="darker_gray_mat" class="visual" />
+                <geom mesh="link5_5" material="gray_mat" class="visual" />
+                <geom mesh="link5_6" material="light_gray_mat" class="visual" />
+                <geom mesh="link5_7" material="gray_mat" class="visual" />
+                <geom mesh="link5_8" material="gray_mat" class="visual" />
+
+                <geom mesh="link5" class="collision" />
+
+                <body name="link6" pos="0 0.091 0.0014165" quat="0 0 -0.707105 -0.707108">
+                  <inertial pos="0.0010312 0.0121761 0.0315481" quat="0.0230402 0.707091 0.0373344 0.70576"
+                    mass="1.2" diaginertia="0.001 0.001 0.001" />
+                  <joint name="joint6" pos="0 0 0" axis="0 0 1" range="-3.14 3.14" />
+                  <geom quat="0.707105 0 0 0.707108" mesh="link6" material="gray_mat" class="visual" />
+                  <geom quat="0.707105 0 0 0.707108" mesh="link6" class="collision" />
+
+                  <geom name="gripper_geom" type="mesh" rgba="0.792157 0.819608 0.933333 0" mesh="link6_1_gripper" />
+                  <site name="attachment_site" size="0.01" pos="0 0 0.1" quat="-1 1 1 1" group="1" />
```

Comment:
I don't think you need an attachment site since this isn't a gripper-less robot right?